htsohm/simulation/surface_area.py

```diff
@@ -66,13 +67,21 @@ def run(run_id, material_id):
     os.makedirs(output_dir, exist_ok=True)
     filename = os.path.join(output_dir, "SurfaceArea.input")
     write_raspa_file(filename, run_id, material_id)
-    print("Calculating surface area of %s-%s..." % (run_id, material_id))
-    sys.stdout.flush()
-    subprocess.run(['simulate', './SurfaceArea.input'], check=True, cwd=output_dir)
+    while True:
+        try:
```

Comment:
Note for the future: this run method is pretty complicated and duplicated across all three simulations. We should probably refactor it out somehow by either making these classes or making a helper method in the future.